src/components/Validator/index.js

```diff
@@ -0,0 +1,96 @@
+import React, { Component } from 'react'
+import helpers from '../../utils/helpers'
+import { PhysicalAddressValue } from '../PhysicalAddressValue'
+import { ValidatorDataPair } from '../ValidatorDataPair'
+import { ValidatorTitle } from '../ValidatorTitle'
+
+class Validator extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      confirmation: null
+    }
+    this.props.metadataContract.getConfirmations({ miningKey: this.props.address }).then(confirmation => {
+      this.setState({ confirmation: confirmation[0] })
+    })
+  }
+
+  render() {
+    let {
+      address,
+      children,
+      contactEmail,
+      createdDate,
+      expirationDate,
+      firstName,
+      index,
+      isCompany,
+      lastName,
+      licenseId,
+      netId,
+      networkBranch,
+      physicalAddresses,
+      updatedDate
+    } = this.props
+
+    if (netId === helpers.netIdByName('dai') && !createdDate) {
```

Comment:
`dai` and other networks names are meet in different places in the code. Let's define constants for them.